session.py

```diff
@@ -0,0 +1,671 @@
+import requests
+
+import json
+
+import toml
+
+from requests.sessions import session
+
+class mySession:
+
+    s = requests.session()
+
+    #baseURL
+    
+    base_URL = "https://wger.de/api/v2/"
+
+
+    #loginURL
+
+    loginURL="https://wger.de/en/user/login"
+
+
+    #baseReferers
+
+    base_workoutReferer = "https://wger.de/en/workout/"
+    base_nutritionReferer = "https://wger.de/en/nutrition/"
+
+
+    def __init__(self):
+        cookies = self.login_and_cookies()
+        
+        #extract csrftoken and sessionId
+        self.csrftoken = cookies[0]
+        self.sessionId = cookies[1]
+
+        #create login header
+        self.headers = {
+        'Content-Type': 'application/json',
+        'Referer': '',
+        'X-CSRFToken' : self.csrftoken,
+        'Authorization' : 'Token bd9ecc45b397f0e22bd0a8bd3304010ab4b586d8',
+        'Cookie':f'csrftoken={self.csrftoken}; sessionid={self.sessionId}'
+    }
+
+    @classmethod
+    def show_request_details(cls, response_list):
+        """ Print the status_code and the content of a request and
```

Comment:
Please use the
"""
docstrings
"""
format, as opposed to the
"""docstring
"""
format. It's easier to read and more aesthetic.